Documentation/ceph-advanced-configuration.md

```diff
@@ -63,6 +64,12 @@ sed -i.bak \
 kubectl apply -f common.yaml -f operator.yaml -f cluster.yaml # add other files as desired for yourconfig
 ```
 
+## Configuration using environment variables
```

Comment:
This doc is intended for advanced configurations, and adding it to the common issues doc seems sufficient. How about if we revert the changes in this doc?